code/modules/projectiles/guns/projectile/shotgun/rushing_bull.dm

```diff
@@ -22,9 +22,9 @@
 	bulletinsert_sound 	= 'sound/weapons/guns/interact/shotgun_insert.ogg'
 	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 10)
 	price_tag = 600
-	penetration_multiplier = 2
+	penetration_multiplier = 1.15 //2 ?????????????????????????????????????
```

Comment:
Its a AP shotgun to be viable for blackshields pvp and heavy armor, keep it to

Reply:
yea nah, we're not doing this without a good reason.